nbgrader/exchange/fetchfeedback.py

```diff
@@ -0,0 +1,66 @@
+import os
```

Comment:
Maybe this file could be called `fetch_feedback.py`, to be consistent with the name of the nbgrader command?

Reply:
yes